.kiro/specs/auterity-ai-platform-unified/templates/compliance-aware-agent-template.py

```
@@ -0,0 +1,676 @@
+            # Log successful completion
+            await self.audit_logger.log_execution_complete(
+                execution_id=execution_id,
+                result=result,
+                audit_result=audit_result
+            )
+            
+            # Learn from execution if enabled
+            if self.learning_enabled:
+                await self.learn_from_compliance_execution(task, result, audit_result)
+            
+            return result
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)